src/views/publish/start.ejs

```diff
@@ -17,4 +18,4 @@
 	</main>
 </div>
 
-<%- include("../partials/bottom"); %>
\ No newline at end of file
+<%- include("../partials/footer"); %>
```

Comment:
Looks like this file needs a newline adding at the end - maybe your new editor doesn't have the newline on save option turned on?

Reply:
Added the new line... No typically I don't have the add a newline option turned on as I've found there are more times than not that I don't want/need a new line at the end of a file.